src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -67,12 +90,31 @@ export function useRefetchQuoteCallback() {
           fee
         })
       } catch (error) {
-        console.error('Error getting the quote (price/fee)', error)
+        const errorType = error?.errorType
+        console.error('Error getting the quote (price/fee):', errorType)
```

Comment:
I would not console log this if it's being handled.

Just log in the else of `if (errorType === ApiErrorCodes.UnsupportedToken)`

Reply:
👍